app/models/cleaner.rb

```diff
@@ -1,2 +1,13 @@
 class Cleaner < ActiveRecord::Base
+  has_many :bookings
+  has_and_belongs_to_many :cities
+  has_many :customers,through: :bookings
+
+  # valition for Cleaner data in model
+
+  validates_presence_of :email,:last_name,:first_name,:quality_score
+  validates :email, format: { with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/, message: ': Invalid Email Address'}
```

Comment:
Use constants EMAIL_REGEX = /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/ for all the constant variables.